pyk/src/pyk/proof/proof.py

```diff
@@ -76,7 +76,9 @@ def __init__(
             ensure_dir_path(self.proof_dir)
 
     @abstractmethod
-    def commit(self, result: SR) -> None: ...
+    def commit(self, result: SR) -> None:
+        """Applies the step result of type `SR` to `self`, modifying `self`."""
```

Comment:
```suggestion
        """Apply the step result of type `SR` to `self`, modifying `self`."""
```

(Example: https://peps.python.org/pep-0257/#one-line-docstrings)

Reply:
Fixed.